jsconfig.json

```diff
@@ -5,7 +5,7 @@
     "checkJs": true,
     "esModuleInterop": true,
     "moduleResolution": "node",
-    "module": "ES2020",
+    "module": "es2022",
```

Comment:
No objections, just curious... What's the ES2022-specific introduction?

Reply:
Dynamic imports with top level await.

```js
const { authConfig } = await import('./components/auth.js');
```

That would have been flagged without es2022.